packages/jsii-reflect/lib/source.ts

```diff
@@ -0,0 +1,49 @@
+import { Assembly } from "./assembly";
+
+export interface SourceLocation {
```

Comment:
Lacks documentation.

Reply:
As does most everything in `jsii-reflect`. But mmkay.